### DISCUSSION TITLE
Random accounts spam

### DISCUSSION THREAD

**Discussion by Sybrenb** [PrestaShop/PrestaShop#34417] [Q&A] [answered]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

Bots create random accounts on our website out of nowhere. How can we solve this?

<img width="763" alt="Scherm­afbeelding 2023-10-27 om 15 40 10" src="https://github.com/PrestaShop/PrestaShop/assets/21313464/8ba1cfae-abea-4927-aebc-26b2765fbc31">

### Steps to reproduce

Spam

### PrestaShop version(s) where the bug happened

1.7.8.7

**Comment by AureRita** [comment]
HI @Sybrenb ,

Thank you for your report but currenly we can"t reproduce it, can you give us more information ? What I have to spam ? What did you do before this happens ? can you give us a video ? some real steps to reproduce ? The php version ? 

You don't know how to get this information? Please read the following article: [How to create a bug report](http://build.prestashop.com/howtos/misc/how-to-create-bug-report/).

Waiting for your feedback

**Comment by Sybrenb** [comment]
Can't reproduce this, we never had this spam but suddenly since last week random accounts get created on the following form (create account):

<img width="444" alt="Scherm­afbeelding 2023-10-27 om 16 55 48" src="https://github.com/PrestaShop/PrestaShop/assets/21313464/a3689f90-b5c0-458f-958c-6b785c035970">

php 7.4

**Comment by matks** [comment, marked as answer]
@Sybrenb I suggest you read https://build.prestashop-project.org/news/2018/fighting-against-spamming/ and https://build.prestashop-project.org/news/2019/fighting-against-spamming-again/

**Comment by ash-f** [comment]
I would suggest to try using a Recaptcha module.
PrestaHero has free one.
